cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/executor/ExecutionTaskPlanner.java

```diff
@@ -46,7 +46,7 @@
 public class ExecutionTaskPlanner {
   private final Logger LOG = LoggerFactory.getLogger(ExecutionTaskPlanner.class);
   private final Map<Integer, Map<Long, ExecutionTask>> _partMoveProposalByBrokerId;
-  private volatile long _remainingDataToMove;
+  private long _remainingDataToMove;
   private final Set<ExecutionTask> _remainingReplicaMovements;
   private final Map<Long, ExecutionTask> _leaderMovements;
   private final AtomicLong _executionId;
```

Comment:
`AtomicLong` --> `Long` -- i.e. `ExecutionTaskPlanner` is guaranteed to be accessed by a single thread.